Makefile.in

```diff
@@ -623,6 +623,9 @@ endif
 	@for f in project/cmake/scripts/common/*.cmake; do \
 	  install -m 0644 $$f $(DESTDIR)$(libdir)/@APP_NAME_LC@; \
 	done
+ifeq ($(findstring linux,@ARCH@),linux)
```

Comment:
yup. should be fine

however, is there any good reason for not *always* installing UseMultiArch.cmake ?